src/app/package.json

```diff
@@ -6,21 +6,25 @@
     "dependencies": {
         "@google/markerclusterer": "1.0.3",
         "@material-ui/core": "3.1.0",
+        "@hot-loader/react-dom": "16.8.6",
         "@material-ui/icons": "3.0.1",
         "@turf/turf": "5.1.6",
         "@turf/distance": "6.0.1",
         "axios": "0.19.0",
         "core-js": "2.6.4",
         "file-saver": "2.0.0",
         "google-map-react": "1.1.2",
+        "http-proxy-middleware": "0.19.1",
         "immutability-helper": "2.9.0",
         "lodash": "4.17.11",
         "moment": "2.24.0",
         "prop-types": "15.6.2",
         "react": "16.8.6",
+        "react-app-rewire-hot-loader": "2.0.1",
+        "react-app-rewired": "2.1.3",
         "react-copy-to-clipboard": "5.0.1",
         "react-dom": "16.8.6",
-        "react-map-gl": "3.2.8",
```

Comment:
Is the removal of `react-map-gl` intentional?

Reply:
Yeah -- it is d13e49e5aee6d50d98f2b901be94be3f92243416

react-map-gl was used for the old Mapbox map but since we no longer have that we no longer need the library. I removed it here because I saw it but kept it to a separate commit.